src/main/java/org/spongepowered/mod/SpongeMod.java

```diff
@@ -134,10 +107,25 @@ public void onPreInit(FMLPreInitializationEvent e) {
     @Subscribe
     public void onInitialization(FMLInitializationEvent e) {
         this.registry.init();
+        try {
+            getGame().getServiceManager().setProvider(this, CommandService.class, new SimpleCommandService(getGame().getPluginManager()));
```

Comment:
It isn't possible to get a `PluginContainer` from a `SpongeMod` instance.

See https://github.com/SpongePowered/SpongeAPI/issues/426